packages/core/src/sass/account-wizard.scss

```diff
@@ -147,51 +147,6 @@
             }
         }
     }
-    &__btn-area {
-        display: flex;
-        align-items: center;
-        justify-content: flex-end;
-        align-self: flex-end;
-        padding-top: 2.4rem;
-        width: 100%;
-
-        .dc-btn:not(:last-child) {
-            margin: 0 1rem;
-        }
-        .dc-btn:last-child {
-            margin: 0;
-        }
-        &--absolute {
-            bottom: 0;
-            position: absolute;
-            left: 0;
-            padding: 2.4rem;
-            border-top: 1px solid var(--border-disabled);
-        }
-        &--center {
-            justify-content: center;
-            width: 100%;
-
-            & .dc-btn {
-                min-width: 9rem;
-            }
-        }
-        &--error {
-            @include typeface(--paragraph-left-normal-black);
-            text-transform: none;
-            color: var(--brand-red-coral) !important;
-        }
-        @include mobile {
```

Comment:
Is mobile already handled after moving the account-wizard `ButtonArea`?

Reply:
No It wasn't. Nice catch sir.